src/datanode/src/region_server.rs

```diff
@@ -528,6 +536,37 @@ impl RegionServerInner {
                 self.event_listener.on_region_deregistered(region_id);
             }
         }
+        Ok(())
+    }
+
+    async fn register_logical_regions(
```

Comment:
I'm wondering if it's possible to merge this logic with region open action, but it also works now. We can consider this problem later